media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -1960,6 +1991,7 @@ VAStatus MediaLibvaCaps::CreateDecConfig(
                 && decAttributes[1].value == m_decConfigs[i].m_encryptType
                 && decAttributes[2].value == m_decConfigs[i].m_processType)
         {
+            m_decConfigs[i].m_priority = decAttributes[3].value - CONTEXT_PRIORITY_MAX/2;
```

Comment:
need to add check about decAttributes[3].value. is it valid.

Reply:
> hi @hongbolv , 1, please add VA VERSION CHECK , refer #1012
Done in the PR:https://github.com/intel/media-driver/pull/1020
> 2. if possible , could you split it to several commit? such as: step1. MOS change, step2 DDI set change, step 3. Capability change?.
Done in the PR: https://github.com/intel/media-driver/pull/1020